src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -231,6 +208,16 @@ private List<Autodesk.Revit.DB.ElementId> GetPurgeableElementIds(List<Performanc
             return null;
         }
 
+        private static List<PerformanceAdviserRuleId> GetPerfromanceAdviserRuleId()
```

Comment:
add `ForPurge` to method name at the end, so it's clear which one we're returning